src/validateEmail.test.js

```diff
@@ -8,12 +8,50 @@ describe(`Function 'validateEmail':`, () => {
   });
 
   it(`should return boolean`, () => {
-
+    expect(typeof validateEmail('')).toBe('boolean');
   });
 
   it(`should return 'true' for the valid email...`, () => {
+    expect(validateEmail('zuzu@gmail.com')).toBe(true);
+  });
+
+  it(`should return 'false' if personal_info contains cyryllic letters`, () => {
+    expect(validateEmail('мама@gmail.com')).toBe(false);
+  });
+
+  it(`should return 'true' if personal_info contains digits`, () => {
+    expect(validateEmail('52875@gmail.com')).toBe(true);
+  });
+
+  it(`should return 'true' if personal_info contains special symbols`, () => {
+    expect(validateEmail('ab-8cd@gmail.com')).toBe(true);
+  });
+
+  it(`should return 'false' if first symbol of personal_info is .`, () => {
+    expect(validateEmail('.abcde@gmail.com')).toBe(false);
+  });
 
+  it(`should return 'false' if last symbol of personal_info is .`, () => {
+    expect(validateEmail('abcde.@gmail.com')).toBe(false);
   });
 
-  // write more tests here
+  it(`should return 'false' if there are .. inside personal_info`, () => {
+    expect(validateEmail('ab..cde@gmail.com')).toBe(false);
+  });
+
+  it(`should return 'true' if there are . inside personal_info`, () => {
```

Comment:
You can use one or two cases with the result 'true' instead of four cases.
For example:
1. zuzuмамаab-8cdab.cde@gmail.com 
or
1.  zuzuмамаab@gmail.com
2.  ab-8cdab.cde@gmail.com

Reply:
в такому випадку ми не дотримуємося ізольованості тестів. якщо впаде тест з 4 первевірками - невідомо яка перевірка не пройшла.